docs/install-azuredisk-csi-driver.md

```diff
@@ -1,41 +0,0 @@
-# Install azuredisk CSI driver on a kubernetes cluster
```

Comment:
Also fix the links from README.md, since the file is deleted.

Reply:
I reverted the file name change.